Cabal-syntax/Cabal-syntax.cabal

```diff
@@ -1,4 +1,4 @@
-cabal-version: 2.0
+cabal-version: 2.2
```

Comment:
Isn't this outside our support window?

Reply:
_I mean_ our support window includes GHCs that won't be able to build this

Reply:
Yeah, looks like GHC 8.0 includes Cabal 1.24, which won't be able to parse cabal-version 2.2, **or even 2.0**.

Reply:
https://hackage.haskell.org/package/Cabal-2.0.1.1, the last Cabal before 2.2, has been released on 2017-12-04, which is 4.5 years ago.

Reply:
> https://hackage.haskell.org/package/Cabal-2.0.1.1, the last Cabal before 2.2, has been released on 2017-12-04, which is 4.5 years ago.

That isn't enough. According to CONTRIBUTING.md if we want to support GHC 8.0 we have to support the Cabal included in that release, which is 1.24

Reply:
@fgaz: well spotted. Fixed. Not fixed on master because, possibly 3.10 will be released when 2.2 is legal and so we can keep correct names of licences, etc.

Reply:
@fgaz: apologies, github was not showing your comments except the first one, so I basically repeated all you said, ignoring you completely. :D